test/blueprints/Do.Test.Blueprints.Service.Test/Orm/QueryingEntities.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Do.Orm;
+
+namespace Do.Test.Orm;
+
+public class QueryingEntities : TestServiceSpec
```

Comment:
:thinking: these don't look like functional tests, more like unit tests. we should discuss whether to test over entities class or not